cle/backends/macho/macho.py

```diff
@@ -331,6 +331,10 @@ def _parse_load_commands(self, lc_offset):
                 f"Assertion triggered: {count} < {self.ncmds} or {offset - lc_offset} < {self.sizeofcmds}"
             )
 
+    @staticmethod
+    def extract_soname(path):
```

Comment:
`os.path.basename(path)`?

Reply:
doesn't that use os.pathsep which varies by platform?

Reply:
Oh right, good call